database/itemModel.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+mongoose.connect('mongodb://localhost:27017/recommended', { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
you don't actually need to add the default port, it should default to 27017 automatically.